docs/configuration.rst

```diff
@@ -284,10 +98,10 @@ Removes a given reference set from a given data repository.
     $ ga4gh_repo remove-referenceset path/to/datarepo aReferenceSet
 
 ++++++++++++++++
-add-ontologymap
+add-ontology
```

Comment:
Yeah, I've changed it to ``ontology`` at this level, in an attempt to future-proof the command a bit. See #1117.